src/global_styling/functions/_colors.ts

```diff
@@ -28,10 +27,79 @@ export const tint = (color: string, ratio: number) =>
 export const shade = (color: string, ratio: number) =>
   chroma.mix(color, '#000', ratio).hex();
 
-// TODO
-export const makeHighContrastColor = (color: string) => color;
-// TODO
-export const makeDisabledContrastColor = (color: string) => color;
+export const saturate = (color: string, amount: number) =>
+  chroma(color).set('hsl.s', `+${amount}`).css();
+
+// Given a $foreground and a $background, make the $foreground AA accessibility by slightly
+// adjusting it till the contrast is high enough
```

Comment:
I'd love it if we could use the pre-fab style of function comments with the `@param` and stuff similar to our services. This will make it easier to parse for docs later.

Reply:
But this is **certainly** where I'd love your help is in converting all our color functions.

Reply:
> I'd love it if we could use the pre-fab style of function comments with the @param and stuff

Yep, we can do that.

> converting all our color functions

Yep, I'm realizing just how much work there is to be done to get to parity with Sass. I'll focus here.

Reply:
Sweet. You can also probably just put them in the normal `services` or `utilities` folders alongside our other JS color functions too.

Reply:
Ha! I just saw your last commit did this 🥇